src/ess/reduce/nexus.py

```diff
@@ -234,7 +239,15 @@ def _load_group_with_positions(
         loaded = cast(
             sc.DataGroup, _unique_child_group(instrument, nx_class, group_name)[()]
         )
-        loaded = snx.compute_positions(loaded)
+
+        transform_out_name = 'transformation'
```

Comment:
I think we were using `transform` in ESSsans? `transformation` is nearly identical to `transformations`, the current name used for the subgroup holding the transformations, so this would be quite confusing. Can we stick with `transform`?